pkg/controllers/monitoring-stack/controller.go

```diff
@@ -100,7 +110,12 @@ func RegisterWithManager(mgr ctrl.Manager, opts Options) error {
 		Owns(&rbacv1.RoleBinding{}).WithEventFilter(p).
 		Owns(&monv1.ServiceMonitor{}).WithEventFilter(p).
 		Owns(&policyv1.PodDisruptionBudget{}).WithEventFilter(p).
-		Complete(r)
+		Watches(&source.Kind{Type: &grafanav1alpha1.GrafanaDataSource{}}, handler.EnqueueRequestsFromMapFunc(func(object client.Object) []reconcile.Request {
+			name := object.GetAnnotations()[grafanaDatasourceOwnerName]
+			namespace := object.GetAnnotations()[grafanaDatasourceOwnerNamespace]
+			namespaceName := types.NamespacedName{Name: name, Namespace: namespace}
+			return []reconcile.Request{{NamespacedName: namespaceName}}
```

Comment:
nit: `namespace*d*Name` to match `types.Namespace*d*Name`

Reply:
isnt namespaceName a better name for a variable?

Reply:
its a `nit` so please feel free to ignore it ... I wouldn't mind if the var was called `nn` :D 
but the point I was trying to make was consistency. 

IMHO  `sheep := Sheep()` is better than `shep := Sheep()` which is better than  ` sheep := Dog()`

Reply:
Done